Makefile

```diff
@@ -747,6 +750,11 @@ npm-cache:
 	touch -c node_modules
 
 node_modules:
+ifneq ($(NODEJS),)
+ifneq ($(NODE_VERSION),$(shell $(NODEJS) --version | sed 's/\(v[0-9]*\.[0-9]*\)\.[0-9]*/\1/'))
```

Comment:
I don't know why you don't my `$(NODEJS) --version | awk -F. '{ print $1 FS $2 }'` :) This works very well and is more readable